src/api/local-storage-provider.model.ts

```diff
@@ -34,34 +35,58 @@ export class LocalStorageProvider extends PersistenceStoreProvider {
     this.storage = new PouchDB(DATABASE_NAME);
   }
 
-  async saveMonthRevision(type: RevisionType, monthDataModel: MonthDataModel): Promise<void> {
+  async saveBothMonthRevisionsIfNeeded(
```

Comment:
W metodzie chodzi o to, żeby jeśli jest taka potrzeba zapisać obie rewizje grafiku. Potrzeba pojawia się wtedy, gdy wersja przeciwna jest pusta, wtedy propagujemy zapis także na tę przeciwną rewizję.